sys/net/gnrc/network_layer/ipv6/static_addr/gnrc_ipv6_static_addr.c

```diff
@@ -117,6 +117,10 @@ void auto_init_gnrc_ipv6_static_addr(void)
     gnrc_netif_t *upstream = NULL;
     gnrc_netif_t *downstream = NULL;
 
+    if (gnrc_netif_numof() == 1) {
+        upstream = gnrc_netif_iter(NULL);
+    }
+
```

Comment:
Use `gnrc_netif_highlander()` here, to optimize here, too, so all the searching below can be optimized away?
```suggestion
    if (gnrc_netif_highlander() || gnrc_netif_numof() == 1) {
        upstream = gnrc_netif_iter(NULL);
        /* ... */
    }
    else {
        /* ... */
    }
```

Reply:
I somehow thought that was already part of `gnrc_netif_numof()`